tcms/rpc/tests/test_testrun.py

```diff
@@ -425,6 +430,10 @@ def verify_api_with_permission(self):
         self.assertEqual(result["plan"], self.plan.pk)
         self.assertEqual(result["build"], self.build.pk)
         self.assertEqual(result["manager"], self.test_run_fields["manager"])
+        self.assertEqual(result["start_date"], self.test_run_fields["start_date"])
+        self.assertEqual(result["stop_date"], self.test_run_fields["stop_date"])
+        self.assertEqual(result["planned_start"], self.test_run_fields["planned_start"])
+        self.assertEqual(result["planned_stop"], self.test_run_fields["planned_stop"])
```

Comment:
These assertions will fail because the data is in 2 different formats/data types. On the left you have a result which Python serializes/deserializes as a DateTimeObject while on the right you have a string. 

See the previous changes you added. They compare the result with an object from the database where both are the same data type.